src/include/firebird/impl/blr.h

```diff
@@ -456,4 +459,6 @@
 #define blr_outer_map_message		(unsigned char) 1
 #define blr_outer_map_variable			(unsigned char) 2
 
+#define blr_skip_locked		(unsigned char) 222
```

Comment:
BLR codes are limited and we close to the limit. 
Could `blr_skip_locked` be implemented as sub-code of `blr_writelock` ?

Reply:
> Could `blr_skip_locked` be implemented as sub-code of `blr_writelock` ?

I don't think so.

Reply:
As SKIP LOCKED was suggested also for UPDATE/DELETE, this is not going to be possible. Perhaps it could be done using blr_marks, but personally I don't like it much.
We're not that close to the limit yet, as we have also gaps in numbering to be reused. And we may reserve the last BLR verb as a prefix for the second level of numbering (and use USHORT for BLR verbs internally).

Reply:
Agree.